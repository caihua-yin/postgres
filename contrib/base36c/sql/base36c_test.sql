create extension base36c;
select base36_encode_c(0);
select base36_encode_c(1);
select base36_encode_c(10);
select base36_encode_c(35);
select base36_encode_c(36);
select base36_encode_c(123456789);
