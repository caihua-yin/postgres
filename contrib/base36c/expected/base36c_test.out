create extension base36c;
select base36_encode_c(0);
 base36_encode_c 
-----------------
 0
(1 row)

select base36_encode_c(1);
 base36_encode_c 
-----------------
 1
(1 row)

select base36_encode_c(10);
 base36_encode_c 
-----------------
 a
(1 row)

select base36_encode_c(35);
 base36_encode_c 
-----------------
 z
(1 row)

select base36_encode_c(36);
 base36_encode_c 
-----------------
 10
(1 row)

select base36_encode_c(123456789);
 base36_encode_c 
-----------------
 21i3v9
(1 row)

